### ISSUE TITLE
how to delete channel

### ISSUE THREAD

**Issue by soxft** [mattermost/mattermost-server#19830]
when i use` mmctl channel delete team:off-topic` to delete a channel

it tells me that `Unable to delete channel 'off-topic' error: : The permanent channel deletion function is disabled. Contact your system administrator。, `

But I can't find this button on the system console

**Comment by amyblais** [comment]
@soxft 
1. What Mattermost server version are you on?
2. What OS and version are you using?
3. Do you see any additional Mattermost server log errors?

**Comment by soxft** [comment]
> @soxft
> 
> 1. What Mattermost server version are you on?
> 2. What OS and version are you using?
> 3. Do you see any additional Mattermost server log errors?

1. I am using mattermost 6.5.0
2. the system is  ubuntu 20.04
3. no errors about delete channel in /opt/mattermost/logs/mattermost.log

**Comment by amyblais** [comment]
@agnivade @isacikgoz Would this be a bug?

**Comment by isacikgoz** [comment]
Nope, this is not a bug, the value of `ServiceSettings.EnableAPIChannelDeletion` needs to be changed from config file IIRC.

**Comment by isacikgoz** [comment]
Anyway, created a ticket to improve the UX here: https://mattermost.atlassian.net/browse/MM-42819

**Comment by soxft** [comment]
that's works, thanks for help
